### DISCUSSION TITLE
Starting the Dev discussions.

### DISCUSSION THREAD

**Discussion by IslandTrish** [OpenHoldem/openholdembot#44] [Development]
Welcome to the development section!

**Comment by IslandTrish** [comment]
Hi Mudrox,

I tried to add you as an owner, but it seems your 2FA isn't setup correctly?

**Comment by SalemMaxInMontreal** [comment]
Hi, I know this is not the right place to ask, but is it the forum down?

**Reply by IslandTrish** [reply]
Hi Salem,
Yes this is a good place to ask, I activated Discussions here to facilitate people who are unable to contact us on the MIM forum.

We are working on this issue and will resolve it soon.

I sent an email to the email address you registered from on MIM to explain better, if you didn't receive that email (or you have changed your email since) please feel free to contact us here again.

Regards
Trish
